### PR TITLE
Support crazyflie

### PR DESCRIPTION
**Pull request by aftersomemath**
Add crazyflie as a target. Support new sensors way of choosing altimeters. Move camera launching into hardware section since cameras are hardware.

### CODE REVIEW

File: launch/base_hardware.launch [hunk 1]
```diff
@@ -33,6 +43,35 @@
...
+        <include file="$(find iarc7_launch)/launch/static_transforms_crazyflie.launch" />
+
+        <!-- a fake roombas for motion coordinator -->
+        <node name="fake_roombas"
+            pkg="rostopic"
+            type="rostopic"
+            args="pub -s /roombas iarc7_msgs/OdometryArray
+                    &quot;{}&quot;
+                    -r 1" />
+        <!-- endif -->
```
Comment (amiller27) [inline launch/base_hardware.launch]: This comment shouldn't be here

File: launch/base_hardware.launch [hunk 2]
```diff
@@ -5,6 +5,10 @@
     <!-- if we're in the sim -->
     <group if="$(eval str(arg('platform')) == 'sim')" >
         <include file="$(find iarc7_simulator)/launch/morse.launch" />
+
+        <include file="$(find iarc7_vision)/launch/vision_node.launch">
```
Comment (amiller27) [inline launch/base_hardware.launch]: This really shouldn't be in base_hardware

File: launch/base_hardware.launch (hunk 1 above)
Comment (aftersomemath) [reply launch/base_hardware.launch]: Fixed

File: launch/base_hardware.launch (hunk 2 above)
Comment (aftersomemath) [reply launch/base_hardware.launch]: Yeah it's not hardware.

**Comment by aftersomemath** [comment]
Significant change, I allowed for different process noise covariance and initial noise covariance for different platforms.

**Comment by amiller27** [comment]
Why do we need those to be different?  In theory they should be the same for all platforms.  Differences between platforms should just be in the variances of the sensors on that platform.  Do we currently need different numbers for the crazyflie?

**Comment by aftersomemath** [comment]
I'm not sure yet. Don't merge this PR. I'm having a bunch of trouble with the IMU on the crazyflie. The filtered result sometimes isn't right when compared to the z ranger. It will say we are accelerating downwards when the drone is shooting towards the ceiling. I initially made these changes last night before I realized that this was the problem.

**Comment by aftersomemath** [comment]
The separate configurations was not needed. I needed to change the propellers...... Too much vibration.

Ready.

**Review by amiller27 — APPROVED** [review]

**Pull request merged**